apps/api-extractor/src/collector/Collector.ts

```diff
@@ -43,6 +43,29 @@ export interface ICollectorOptions {
   extractorConfig: ExtractorConfig;
 }
 
+/**
+ * Resolve the name of a file to an absolute path relative to a TypeScript SourceFile node.
+ */
+function resolveNameRelativeToSourceFile(name: string, source: ts.SourceFile): string {
```

Comment:
Can you simply use [path.relative()](https://nodejs.org/api/path.html#path_path_relative_from_to)?

Reply:
For this one, I don't think so. I need to boot into Windows and check it, but I believe that the reference directive in text has to follow the POSIX convention and use forward slash as its path separator just like a node module, but the file name of the source file could be in Windows convention, so relying on `path` to process the reference path may not be safe. Need to do some testing.

Reply:
If the input and output paths are POSIX format, then you can use `path.posix.relative()`.

If the input path is Windows style and you want to convert to POSIX, there is no theoretically correct API for that, but `Text.replaceAll(thePath, '\\', '/')` works reasonably well with Node.js